File: 8-kyu/find-maximum-and-minimum-values-of-a-list/c++/solution.cpp
#include <vector>
using namespace std;

int min(vector<int> list){
    
    auto min = list[0];
    for (int n : list) {
        if (n < min) min = n;
    }
    return min;
}

int max(vector<int> list){
    
    auto max = list[0];
    for (int n : list) {
        if (n > max) max = n;
    }
    return max;
}

File: 8-kyu/find-maximum-and-minimum-values-of-a-list/javascript/solution.js
#include <algorithm>
#include <vector>
using namespace std;

int min(vector<int> list)
{
    sort(list.begin(), list.end());
    return list.front();
}

int max(vector<int> list)
{
    sort(list.begin(), list.end());
    return list.back();
}